packages/types/src/data-types.ts

```diff
@@ -13,6 +13,22 @@ export type AppInstanceInfo = {
   proposedByIdentifier: string; // xpub
   proposedToIdentifier: string; // xpub
   intermediaries?: string[];
+
```

Comment:
So they _are_ defined here. Why not use them across the other packages then?

Reply:
This is the data type for the client side object. I suppose it could be getting re-used.

Reply:
Added.